app/models/page.rb

```diff
@@ -59,7 +59,11 @@ def image_path(name)
     end
 
     def paginated_resource_docs_url
-      @url_helpers.docs_path + '/rest-api#pagination'
+      @url_helpers.docs_path + "/rest-api#page-number-pagination"
```

Comment:
This URL is missing the H2 reference:

```suggestion
      @url_helpers.docs_path + "/rest-api#pagination-page-number-pagination"
```